VideoWeb/VideoWeb/ClientApp/src/styles.scss

```
@@ -144,6 +144,10 @@ $govuk-global-styles: true;
   transition: visibility 0s linear 0s, opacity 300ms;
 }
 
+.wrap-anywhere {
+  overflow-wrap: anywhere;
```

This is not supported by Safari. 

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#browser_compatibility
```suggestion
  overflow-wrap: break-word;
  overflow-wrap: anywhere;
```

Perhaps a different name as  well